ogcapi-features/README.md

```diff
@@ -157,10 +157,11 @@ The following Link relations must exist in the Item object returned from the `/c
 | **rel**  | **href**                                     | **Media Type**       | **From**            | **Description**                                     |
 | -------- | -------------------------------------------- | -------------------- | ------------------- | --------------------------------------------------- |
 | `root`   | `/`                                          | application/json     | STAC API - Features | The root URI                                        |
+| `collection` | `/collections/{collectionId}`            | application/json     | OAFeat              | The containing Collection |
```

Comment:
is the `collection` link rel defined and/or required by OAFeat, or only STAC Item?

Reply:
The latest commit updates this to be "STAC Item" instead of "OAFeat."

I don't see "parent" in either OGC API – Features or Common.  Should the "From" cell for "parent" also be updated to point to the STAC spec?